src/test/java/emissary/core/InterruptibleCharSequenceTest.java

```diff
@@ -77,6 +77,7 @@ void testNoninterruptibleString() throws InterruptedException {
     @Test
     void testInterruptibleCharSequence() throws InterruptedException {
         long sleepMillis = 32;
+        String INPUT = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaab";
```

Comment:
Recommend changing to a local variable name to not shadow the static class variable.

Reply:
Done